docs/sources/operations/storage/legacy-storage.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Legacy storage
+menuTitle:
+description: Describes deprecated legacy storage options for Loki that are superseded by single store.
+weight: 1000
+---
+# Legacy storage
+
```

Comment:
This may be your plan for your next PR, but a lot of this content looks like it already exists in https://github.com/grafana/loki/blob/main/docs/sources/operations/storage/_index.md, so we should probably delete it from that topic.

Reply:
Definitely. My idea was to have a separate place/page for legacy storage, so we don't need to mix recommended and deprecated content.